controllers/rhmi/rhmi_controller.go

```diff
@@ -288,6 +288,7 @@ func (r *RHMIReconciler) Reconcile(request ctrl.Request) (ctrl.Result, error) {
 		})
 	}
 	metrics.SetRHMIStatus(installation)
+	metrics.SetRhmiVersions(string(installation.Status.Stage), installation.Status.Version, installation.Status.ToVersion, installation.CreationTimestamp.Unix())
```

Comment:
hey @austincunningham just a question, why is this required? 
We are currently setting the metric on 
a) first reconcile
b) when stage reports as "completed"
The description on the b) check is `Check for stage complete to avoid setting the metric when installation is happening` - is there a reason why we want to avoid setting the metric? If not, I would suggest removing the logic around setting the metric when the "completed" stage is reported, if we are setting the metric every time anyway. 

IMO I don't think we have to set metrics on every reconcile but instead, on first reconcile + after completed is reported (unless I'm missing something!)

Reply:
@MStokluska the alert was currently checking for the absent stage="completed", but the absent of the metric would trigger this alert as well.

Reply:
Looks like I was miss interpreting the data as clock on cluster was an hour behind

![image](https://user-images.githubusercontent.com/16667688/118610835-1a68bc80-b7b4-11eb-956c-b2c6a9ed78d7.png)

I can remove this :+1: